pkg/data/envdManaged.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2022 The envd Authors
```

Comment:
The file name does not follow the golang convention, please use envd.go or datasource.go

Reply:
https://medium.com/@kdnotes/golang-naming-rules-and-conventions-8efeecd23b68